<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadView</class>
 <widget class="QWidget" name="UploadView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>上传文件</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="fileLine">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectBtn">
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="uploadBtn">
        <property name="minimumSize">
         <size>
          <width>251</width>
          <height>71</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>251</width>
          <height>71</height>
         </size>
        </property>
        <property name="text">
         <string>上传</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="minimumSize">
         <size>
          <width>361</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>361</width>
          <height>41</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
